packages/core/src/Box/Box.tsx

```diff
@@ -44,7 +44,7 @@ import {
   paletteFamilies,
   roundedValues,
 } from '../utils'
-import { ColorSchemes } from '../theme'
+import { ColorSchemeName } from '../storybook/args'
```

Comment:
In my mind, storybook and tests are like dev deps and shouldn't make their way into source code. Is this an accident or by design?

Reply:
I don't have a better place to put the type we use in the source code since it's derived from an array of values that are only used in stories that live in the same file.

Reply:
@sdalonzo I think it'd make more sense to colocate these values with the `createColorScheme` function now that they're not used solely for story args. Thoughts?

Reply:
they are in both. storybook/args just re-exports for convenience in stories, and i'd only use it for stories.